.devcontainer.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "Home Assistant Frontend",
-  "build": {
-    "dockerfile": "Dockerfile",
-    "context": ".."
-  },
+  "image": "mcr.microsoft.com/devcontainers/python:0-3.10",
```

Comment:
Since this is a frontend repo it might make sense to use node as the base instead of python.
But adding python ontop of node takes around 30s on my machine while adding node ontop of python takes 3s.

Reply:
Python is needed to run core inside the container

Reply:
Yes